tests/scattering1d/test_torch_scattering1d.py

```diff
@@ -436,7 +436,19 @@ def test_Q(device, backend):
     length = 1024
     shape = (length,)
 
+    # test different cases for Q
+    sc_int = Scattering1D(J, shape, Q=(8, ), backend=backend, frontend='torch').to(device)
+
+    sc_tuple = Scattering1D(J, shape, Q=(8, 1), backend=backend, frontend='torch').to(device)
```

Comment:
this is nice but we should do it for the other libraries too?

Reply:
All the frontends have been tested now.